CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [4.8.1]
+
+#### Fixed
+- Improved encrypt/decrypt performance by caching KeyStore, Cipher and Symmetric Key [SDKS-4090]
```

Comment:
I believe we should also mention the new StrongBox config.

Reply:
Updated.